src/agent/onefuzz/src/input_tester.rs

```diff
@@ -147,7 +147,6 @@ impl<'a> Tester<'a> {
             &env,
             self.timeout,
             IGNORE_FIRST_CHANCE_EXCEPTIONS,
-            None,
```

Comment:
The task impls use the `input-tester` crate via this call site, which hardcodes the module cache to `None` (meaning we never used it to record coverage).